pkg/cmd/pulumi/destroy.go

```diff
@@ -393,8 +393,9 @@ func newDestroyCmd() *cobra.Command {
 		"Suppress display of the state permalink")
 	cmd.Flag("suppress-permalink").NoOptDefVal = "false"
 	cmd.PersistentFlags().BoolVar(
-		&continueOnError, "continue-on-error", false,
-		"Continue to perform the destroy operation despite the occurrence of errors")
+		&continueOnError, "continue-on-error", cmdutil.IsTruthy(os.Getenv("PULUMI_CONTINUE_ON_ERROR")),
```

Comment:
My only comment is -- I think this will be fine but it's just worth us double checking -- this environment variable lookup will happen before some of Cobra's initialisation code has run. I think this has bitten us before (though not with an environment variable) when it came to `Interactive`. @julienp does this look OK to you?

Reply:
`IsTruthy` is a pure function on a string. Do you think `os.Getenv` can mess with Cobra somehow?

Reply:
Yes this is fine, it is only a problem when you try to read the flag's variable, `continueOnError` in this case, before `cobra.Run`.